docs/content/v2025.1/yugabyte-platform/manage-deployments/ysql-major-upgrade-yba.md

```diff
@@ -0,0 +1,68 @@
+---
+title: YSQL major upgrade in YugabyteDB Anywhere
+headerTitle: YSQL major upgrade
+linkTitle: YSQL major upgrade
+description: Upgrade YugabyteDB to PostgreSQL 15 version in YBA
+headcontent: Upgrade the YugabyteDB software on your universe to a version that supports PG15
+menu:
+  v2025.1_yugabyte-platform:
+    identifier: ysql-major-upgrade-yba
+    parent: upgrade-software
+    weight: 50
+type: docs
+---
+
+Upgrading YugabyteDB from a version based on PostgreSQL 11 ({{<release "2024.2">}} and earlier) to a version based on PostgreSQL 15 ({{<release "2025.1">}} or later) requires additional steps.
+
+The upgrade is fully online. While the upgrade is in progress, you have full and uninterrupted read and write access to your cluster.
```

Comment:
Need a clarification here - When you say 
"While the upgrade is in progress, you have full and uninterrupted read and write access to your cluster." - This refers to the user permissions right, because in the next section, you mention that DDL statements are blocked during upgrade.